### ISSUE TITLE
Dying of Synthanol (brain damage?) causes hilarious spam.

### ISSUE THREAD

**Issue by Spacemanspark** [ParadiseSS13/Paradise#3695]
**Problem Description**:
Somebody had to do it, so Spark went and got himself killed via Synthanol. Upon dying, Spark started spewing out a constant *deathgasp message.
@FlattestGuitar 

**What did you expect to happen**: 
I expect it to not spam the chat.

**What happened instead**:
It spammed the chat with my death message.

**Why is this bad/What are the consequences:**
It's obnoxious.

**Steps to reproduce the problem**:
-Get drunk on synthanol
-Die of synthanol (brain damage)?
-Get endless deathgasp spam.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: [i.imgur.com/kSSIrVG.png](http://i.imgur.com/kSSIrVG.png)

**Comment by FlattestGuitar** [comment]
You had to drink A TON to actually get that to kill you, so nice work. 

Synthanol doesn't do much else than add brain damage. I believe this is a bug with dying if brain damage, regardless of what caused it.

**Comment by FalseIncarnate** [comment]
Stepping in to add some info (I treated Spark with a bit of adminbuse to fix him up)

Spark had roughly 300 units of synthanol (little under), and 120-ish brain damage. His stat was set to 2, but continued to spam the death message.

If I had to guess, IPC death doesn't play nice with brain damage (as flattest mentioned). I had to use some var-editing to remove a bit of the synthanol and reset your stat to be able to give the liquid solder and antihol we gave you a chance to take effect and repair your damage.

Further proof that drinking kills, right here.
